.github/workflows/release.yml

```diff
@@ -65,6 +84,15 @@ jobs:
         - os: ubuntu-20.04
           target: x86_64-unknown-linux-musl
 
+        - os: aarch64-apple-darwin
```

Comment:
What's the difference between an OS field of "macos-11" and an OS field of "aarch64-apple-darwin"?

Reply:
I renamed it to minimize confusion. It boils down to the labels on GitHub Action Runners used to run the task on the desired runner. `aarch64-apple-darwin` is the self-hosted Apple Silicon (M1) runner, `macos-11` is GitHub-hosted `X86_64` MacOS runner.